.jenkins/pytorch/test.sh

```diff
@@ -23,7 +23,7 @@ export LANG=C.UTF-8
 #       see https://github.com/pytorch/pytorch/issues/60111
 IN_PULL_REQUEST=${CIRCLE_PULL_REQUEST:-}
 
-if [[ "$BUILD_ENVIRONMENT" == *-slow-* ]]; then
+if [[ "$BUILD_ENVIRONMENT" == *-slow-* || $TEST_CONFIG == 'slow' ]]; then
```

Comment:
shellscript noob q:
what's the difference between double quoted variable `"$BLA"`, bracketed `"${BLA}"` and naked `$BLA` here?
i am sensing it might be relate to macos/linux convention but i dunno why there's so much inconsistent in these if statementss

Reply:
there's no difference between using quotes vs not within `[[` double brackets `]]`, which is why shellcheck doesn't complain here; also the curly braces are basically never necessary, so idk why people are using them in general here